apis/python/src/tiledbsoma/_read_iters.py

```diff
@@ -240,9 +240,11 @@ def _reindexed_table_reader(
                 col = tbl.column(f"soma_dim_{d}")
                 if d in self.axes_to_reindex:
                     if d == self.major_axis:
-                        col = pd.Index(coords[self.major_axis]).get_indexer(col.to_numpy())  # type: ignore[no-untyped-call]
+                        col = build_index(coords[self.major_axis]).get_indexer(
```

Comment:
See previous comment about where the concurrency level is found - suggest using the context

Reply:
Sounds good. I will go down through the stack to get the context here.